QueryReplay.py

```diff
@@ -95,16 +101,20 @@ def __getConnection(self, name: str):
   def __closeConnections(self):
     logging.info(datetime.now().strftime("%Y-%m-%d %H:%M:%S") + " - closing connections")
     for conn in self.connections.values():
+      try:
+        conn.cursor().close()
```

Comment:
Otherwise, LGTM

Reply:
Ahaha, the hint in the log message with "cancelling query". All good, thanks!

Reply:
I think I added that code you referenced while I was testing if that would be enough to cancel queries and your gut feeling is correct it doesn't.  I guess I will clean that up to avoid confusion.